api4/bot.go

```diff
@@ -100,7 +100,10 @@ func getBot(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 	botUserId := c.Params.BotUserId
 
-	includeDeleted := r.URL.Query().Get("include_deleted") == "true"
+	includeDeleted := false
```

Comment:
@lieut-data, thank you! I did as you requested. If I may ask, could it be the case that the Debug level could be better suited? Considering this error may be triggered by clients rather than the server itself? Just wondering :)

Reply:
Yes, I think debug should be just fine here, since there's probably no action for the server administrator to take.